packages/main/src/RangeSlider.ts

```diff
@@ -540,6 +547,20 @@ class RangeSlider extends SliderBase implements IFormInputElement {
 		this._endValueAtBeginningOfAction = undefined;
 	}
 
+	_onInputChange(e: Event) {
+		const ctor = this.constructor as typeof RangeSlider;
+
+		const input = e.target as HTMLInputElement;
+		const value = ctor.clipValue(parseFloat(input.value), this._effectiveMin, this._effectiveMax);
```

Comment:
can you just call `SliderBase.clipValue()`

Reply:
I could, I did it for consistency as I saw this is the way Vladi Tasev migrate it to TS in the other instances